cirq-core/cirq/ops/common_gates.py

```diff
@@ -74,8 +74,8 @@ class XPowGate(eigen_gate.EigenGate):
     $$
     e^{i \pi s t}
     \begin{bmatrix}
-      e^{i \pi t /2} \cos(\pi t) & -i e^{i \pi t /2} \sin(\pi t) \\
-      -i e^{i \pi t /2} \sin(\pi t) & e^{i \pi t /2} \cos(\pi t)
+      e^{i \pi t /2} \cos(\pi t /2) & -i e^{i \pi t /2} \sin(\pi t /2) \\
+      -i e^{i \pi t /2} \sin(\pi t /2) & e^{i \pi t /2} \cos(\pi t /2)
```

Comment:
The `e^{i \pi t /2}` factor is on every matrix element, so for readability we should pull it out and combine with `e^{i \pi s t}`. Could you please do this while you're at it? :-)

Reply:
Gladly
I moved the e^{i \pi t /2} factors to the front, as you suggested :-)

Reply:
Thank you!